.github/workflows/integration-tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build plugin (statically)
       run: |
         CGO_ENABLED=0 go build -tags netgo -ldflags '-extldflags "-static"' \
-          -o terraform-provider-conjur .
+          -o /terraform-provider-conjur/dist/terraform-provider-conjur_linux_amd64/terraform-provider-conjur_v0.4.0 .
```

Comment:
should this hardcode the version? can we pull this as a param from the version file?